jackson2/src/test/java/org/jdbi/v3/jackson2/TestJackson2Plugin.java

```diff
@@ -0,0 +1,88 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.jdbi.v3.jackson2;
+
+import java.util.List;
+
+import com.fasterxml.jackson.annotation.JsonCreator;
+import com.fasterxml.jackson.annotation.JsonProperty;
+import com.fasterxml.jackson.databind.ObjectMapper;
+
+import org.assertj.core.groups.Tuple;
+import org.jdbi.v3.core.Handle;
+import org.jdbi.v3.postgres.PostgresDbRule;
+import org.jdbi.v3.sqlobject.statement.SqlQuery;
+import org.jdbi.v3.sqlobject.statement.SqlUpdate;
+import org.jdbi.v3.testing.JdbiRule;
+import org.junit.Before;
+import org.junit.Rule;
+import org.junit.Test;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+public class TestJackson2Plugin {
+// tag::jackson2example[]
```

Comment:
Dropping the whole unit test into the docs doesn't exactly convey how the plugin should be used.

Not a blocker for merge, but I think users would find more value in a conversational "first you add the dep: <example>, then you install the plugin: <example>, then if you annotate your bound properties or SQL objects like this: <example> then code like this Just Works :tm: <example>